MatrixSDK/LocationSharing/MXBeaconInfoSummaryProtocol.swift

```diff
@@ -0,0 +1,44 @@
+// 
+// Copyright 2022 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+
+/// MXBeaconInfoSummaryProtocol decscribes a summary of live location sharing for a user sharing his location in a room.
+/// It aggregates the start live location sharing state event `m.beacon_info` with last live location event `m.beacon`.
+@objc
+public protocol MXBeaconInfoSummaryProtocol: AnyObject {
+    
+    /// First beacon info event id that start live location
+    var identifier: String { get }
```

Comment:
`id` is a reserved keyword in Objective-C normally but it seems ok now. True when the SDK will be iOS 13 min it will be interesting to make it `Identifiable`.

Reply:
Ah this explains a lot, I'd never twigged it was reserved in objc! 😅